### PR TITLE
Add a bio file for John Panelli

### PR DESCRIPTION
**Pull request by johnpanelli**
This Pull Request adds a bio for @johnpanelli, as requested in #19. 

@eabbeintu - This was a tough project, don't you think?
@JenniferLu - Did I spell my name right?

### CODE REVIEW

**Comment by johnpanelli** [comment]
I think I spelled my name right.

File: docs/johnpanelli.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# John Panelli Bio
+
+- ** Name:** John Panelli
```
Comment (johnpanelli) [inline docs/johnpanelli.md:3]: Double check the spelling here.

Comment (eabbeintu) [inline docs/johnpanelli.md:3]: I think Pannelli should have two "n"'s

**Comment by JenniferLu** [comment]
John, is Panelli an Italian last name?

**Pull request merged**